class.jetpack-gutenberg.php

```diff
@@ -200,11 +200,18 @@ public static function enqueue_block_editor_assets() {
 			false
 		);
 
+		// This variable will be used by the blocks script to find parts of a code-split bundle.
```

Comment:
I just noticed we have `Jetpack_Initial_State.pluginBaseUrl: "http://localhost/wp-content/plugins/jetpack"` available via https://github.com/Automattic/jetpack/blob/9d10130c07d05174382ae31918991366c77d5fca/class.jetpack-gutenberg.php#L210-L214

I'm not suggesting to remove these two `wp_localize_script`s here, just bringing it up in case we'd want to consolidate everything into one `Jetpack_Initial_State` and keep global scope cleaner.